pkgs/by-name/ma/manga-ocr/package.nix

```
@@ -8,15 +8,15 @@ with python3Packages;
 
 buildPythonPackage rec {
   pname = "manga-ocr";
-  version = "0.1.12";
+  version = "0.1.13";
   disabled = pythonOlder "3.7";
   format = "pyproject";
```

format is depricated. please change it as suggested
```suggestion
  pyproject = true;
```